ginko/src/dts/project.rs

```diff
@@ -74,11 +74,24 @@ impl ProjectFile {
 #[derive(Default)]
 pub struct Project {
     files: HashMap<PathBuf, ProjectFile>,
+    pub include_paths: Vec<PathBuf>,
     pub severities: SeverityMap,
 }
 
 impl Project {
-    pub fn add_file(&mut self, file_name: PathBuf) -> Result<(), io::Error> {
+    pub fn set_include_paths(
+        &mut self,
+        include_paths: Option<Vec<String>>,
```

Comment:
I think a better API design would be to accept a `Vec<String>` here and do the unwrapping on the caller side (i.e., in the main function)

Reply:
I previously had that logic in `main` but thought that it would deflect from it's primary purpose. But can relocate it 👍